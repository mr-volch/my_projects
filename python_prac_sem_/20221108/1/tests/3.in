c = DivStr("HembatusUmbatusKerunosPertunkos")
print(type(tuple(c//4)[1]))
print(tuple(c//4)[1])
print(tuple(c//4)[1] % 2)
