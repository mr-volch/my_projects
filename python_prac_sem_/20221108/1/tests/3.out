<class '__main__.DivStr'>
sUmbatu
u
